src/runtime_src/xdp/profile/device/xdp_base_device.h

```diff
@@ -58,8 +58,10 @@ class Device
   virtual int getTraceBufferInfo(uint32_t nSamples, uint32_t& traceSamples, uint32_t& traceBufSz) = 0;
```

Comment:
Copyright date (line 2) needs to be updated.  Being that we are AMD, you might wish to add the following line:
`Copyright (C) 2022 Advanced Micro Devices, Inc.`